mappings/net/minecraft/client/render/VertexBuffer.mapping

```diff
@@ -15,40 +25,82 @@ CLASS none/bon net/minecraft/client/render/VertexBuffer
 	FIELD o building Z
 	METHOD <init> (I)V
 		ARG 0 initialSize
+	METHOD a getVertexState ()Lnone/bon$a;
 	METHOD a texture (DD)Lnone/bon;
 		ARG 0 u
 		ARG 1 v
+	METHOD a putPosition (DDD)V
+		ARG 0 x
+		ARG 1 y
+		ARG 2 z
 	METHOD a (FFF)V
 	METHOD a color (FFFF)Lnone/bon;
 		ARG 0 red
 		ARG 1 green
 		ARG 2 blue
 		ARG 3 alpha
+	METHOD a putColorMultipliers (FFFI)V
+		ARG 0 red
+		ARG 1 green
+		ARG 2 blue
+	METHOD a putColor (I)V
+		ARG 0 argb
 	METHOD a texture (II)Lnone/bon;
 		ARG 0 u
 		ARG 1 v
+	METHOD a putBrightness (IIII)V
+	METHOD a putColorRGBA (IIIII)V
+		ARG 0 colorIndex
+		ARG 1 red
+		ARG 2 green
+		ARG 3 blue
+		ARG 4 alpha
 	METHOD a begin (ILnone/bxo;)V
 		ARG 0 drawMode
 		ARG 1 format
+	METHOD a getDistanceSq (Ljava/nio/FloatBuffer;FFFII)F
+		ARG 0 buffer
 	METHOD a restoreState (Lnone/bon$a;)V
+		ARG 0 state
+	METHOD a putVertexData ([I)V
+		ARG 0 vertexData
 	METHOD b clear ()V
 	METHOD b pos (DDD)Lnone/bon;
 		ARG 0 x
 		ARG 1 y
 		ARG 2 z
+	METHOD b putNormal (FFF)V
+		ARG 0 x
+		ARG 1 y
+		ARG 2 z
+	METHOD b putColorRGB (FFFI)V
```

Comment:
How does this differ from the other `putColorRGB` ? The last parameter isn't named.
(And, as per other suggestion, `putColor` ? Same for the others.)